### PR TITLE
updates in pkgbuild

### PR DESCRIPTION
**Pull request by filipe-marques**

### CODE REVIEW

**Comment by philmmanjaro** [comment]
pkgrel should be always **pkgrel=1** if we didn't release it yet as binary ...

**Comment by filipe-marques** [comment]
I fix the typos.

**Comment by filipe-marques** [comment]
@philmmanjaro Hey could you please merge this ?

**Comment by filipe-marques** [comment]
you have test the build of this new package, because it requires devtools, but give error!

File: packnroll/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+pkgname=packnroll
+pkgver=1.0.0
+pkgrel=1
+pkgdesc="It's so easy doing the packaging process !"
+arch=('i686' 'x86_64')
+url="https://github.com/filipe-marques/packnroll"
+license=('GPL')
+depends=('qt5-base' 'dev-tools')
+makedepends=('qt5-tools' 'dev-tools')
```
Comment (Kirek) [inline packnroll/PKGBUILD]: Its devtools not dev-tools

**Comment by filipe-marques** [comment]
my mistake, I will fix that !

**Comment by filipe-marques** [comment]
Done !

**Pull request merged**